corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html

```diff
@@ -9,7 +9,6 @@
         </label>
         <div class="controls" style="padding-top: 5px;">
             <input type="hidden"
-                   name="{{ field.html_name }}"
```

Comment:
this is to prevent from displaying like this:
<img width="357" alt="screen shot 2015-12-02 at 5 04 31 pm" src="https://cloud.githubusercontent.com/assets/1757035/11545721/d67415fe-9916-11e5-858d-852afa35fb61.png">

notifying @biyeun in case this is a problem, but it didn't seem to be with any of the other field types on this page